src/components/menu/submenu-block/submenu-block.component.js

```diff
@@ -2,16 +2,16 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { StyledSubmenuBlock } from './submenu.style';
 
-const SubmenuBlock = ({ children, menuType }) => {
+const SubmenuBlock = React.forwardRef(({ children, menuType }, ref) => {
   return (
-    <StyledSubmenuBlock data-component='submenu-block'>
+    <StyledSubmenuBlock ref={ ref } data-component='submenu-block'>
```

Comment:
Is it correct that `ref` is in the `StyledSubmenuBlock` instead of child?

Reply:
Yeah you are right, the issue was actually that when a `SubmenuBlock` is used it's not being filtered and its children used for the keyboard navigation instead. That's why it throwing an error about a ref being passed to a component that doesn't support `forwardRef` when the tests are run. I'll add some code to add this filter in and removed the ref.